test/product_transform_spec.coffee

```diff
@@ -91,8 +93,9 @@ describe 'Product transforms', ->
       "absolute_url": "/product/vw-kleinbus",
       "identifier": "4XfN",
       "price": "0.00 SEK",
+      "price_tag": priceTag(),
       "price_without_currency": 0,
-      "price_with_currency": "0 SEK",
+      "price_with_currency": priceTag(),
```

Comment:
Isn't it nicer to hard code the expected value here, so that we verify that we've built the `priceTag()` function correctly too?